app/app.go

```diff
@@ -304,43 +309,55 @@ func NewOsmosisApp(
 	app.CrisisKeeper = crisiskeeper.NewKeeper(
 		app.GetSubspace(crisistypes.ModuleName), invCheckPeriod, app.BankKeeper, authtypes.FeeCollectorName,
 	)
-	app.UpgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath)
+	app.UpgradeKeeper = upgradekeeper.NewKeeper(
+		skipUpgradeHeights,
+		keys[upgradetypes.StoreKey],
+		appCodec,
+		homePath,
+		app.BaseApp,
+	)
 
-	// this configures a no-op upgrade handler for the v2 upgrade,
+	// this configures a no-op upgrade handler for the v4 upgrade,
 	// which improves the lockup module's store management.
 	app.UpgradeKeeper.SetUpgradeHandler(
-		"v4", func(ctx sdk.Context, plan upgradetypes.Plan) {
-			// Upgrade all of the lock storages
-			locks, err := app.LockupKeeper.GetLegacyPeriodLocks(ctx)
-			if err != nil {
-				panic(err)
-			}
-			// clear all lockup module locking / unlocking queue items
-			app.LockupKeeper.ClearAllLockRefKeys(ctx)
-			app.LockupKeeper.ClearAllAccumulationStores(ctx)
-
-			// reset all lock and references
-			if err := app.LockupKeeper.ResetAllLocks(ctx, locks); err != nil {
-				panic(err)
-			}
-
-			// configure upgrade for gamm module's pool creation fee param add
-			app.GAMMKeeper.SetParams(ctx, gammtypes.NewParams(sdk.Coins{sdk.NewInt64Coin("uosmo", 1)})) // 1 uOSMO
+		"v4", func(ctx sdk.Context, _plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+			// // Upgrade all of the lock storages
+			// locks, err := app.LockupKeeper.GetLegacyPeriodLocks(ctx)
+			// if err != nil {
+			// 	panic(err)
+			// }
+			// // clear all lockup module locking / unlocking queue items
+			// app.LockupKeeper.ClearAllLockRefKeys(ctx)
+			// app.LockupKeeper.ClearAllAccumulationStores(ctx)
+
+			// // reset all lock and references
+			// if err := app.LockupKeeper.ResetAllLocks(ctx, locks); err != nil {
+			// 	panic(err)
+			// }
+
+			// // configure upgrade for gamm module's pool creation fee param add
+			// app.GAMMKeeper.SetParams(ctx, gammtypes.NewParams(sdk.Coins{sdk.NewInt64Coin("uosmo", 1)})) // 1 uOSMO
 
 			prop12(ctx, app)
```

Comment:
Why is this not commented out?

Reply:
Thanks. Done. The reason it wasn't initially was just due to having to comment out the related test as well. Thats now done!